README.md

```diff
@@ -64,6 +64,7 @@ Target has other properties you can play with
 textAlignement 	/* The text alignement */
 labelWidth      /* The width of the label */
 position 		/* The position of your text around the highlight view */
+arrowStartPosition /* Controlls what possition the orrow starts from, set to nill to match position behaviour */
```

Comment:
Controlls what po`s`ition the `a`rrow starts from, set to `nil` to match position behaviour